css-ui-4/Overview.bs

```diff
@@ -1888,6 +1869,23 @@ Animation type: discrete
 
 </dl>
 
+Some controls have internal parts, or sub controls.
+For example, an <code class="lang-markup">&lt;input type=search></code> element
+could have a clear button to clear the text.
```

Comment:
In this context, "a clear button" might be taken to mean "a button with no background color." Scare quotes would help avoid this ("a "clear" button") as would removing the adjective altogether (which seems acceptable given the phrase is immediately followed by a description of the button's purpose).